.expeditor/scripts/verify/run_cargo_test.sh

```diff
@@ -40,6 +40,8 @@ sudo hab pkg install core/zeromq
 sudo hab pkg install core/protobuf --binlink
 sudo hab pkg install core/rust --binlink
```

Comment:
If we're installing `rustup` and a particular toolchain above, do we need to install `core/rust`?